data_summaries/Orca-Math.json

```diff
@@ -0,0 +1,66 @@
+{
+    "orca-math": {
+        "Unique Dataset Identifier": "orca-math",
+        "Dataset Name": "orca-math",
+        "Paper Title": "Orca-Math: Unlocking the potential of SLMs in Grade School Math",
+        "Dataset URL": "https://huggingface.co/datasets/microsoft/orca-math-word-problems-200k",
+        "GitHub URL": "",
+        "Hugging Face URL": "https://huggingface.co/datasets/microsoft/orca-math-word-problems-200k",
+        "Papers with Code URL": "",
+        "ArXiv URL": "https://arxiv.org/abs/2402.14830",
+        "Semantic Scholar Corpus ID": 267897618,
+        "Collection": "Orca-Math",
+        "Collection URL": "https://huggingface.co/datasets/microsoft/orca-math-word-problems-200k",
+        "Languages": [
+            "English"
+        ],
+        "Task Categories": [
+            "Logical and Mathematical Reasoning",
+            "Logical Reasoning Question Answering"
+        ],
+        "Text Sources": [
+            "arithmetic word problems",
+            "Grade School Math"
```

Comment:
Is this a textbook?

Reply:
We need to do a bit more digging here. It looks like the questions are sourced from several datasets. For each of these, can we trace what sources/models the data was collected from (websites, certain textbooks, or something else)?

"We start by collecting sample math word problems from existing open-source
datasets, namely NumGLUE [26], AddSub [13], ALGES [17], ASDiv [24], DRAW [35],
GSM8k [7], MATHQA [2], MultiArith [32], SingeOP [33], SingleEQ [16], and SVAMP [30].
We collect a total of 36, 217 problems."

Reply:
On it!